index.js

```diff
@@ -0,0 +1,208 @@
+'use strict';
+
+/**
+ * Оголошуємо змінні з HTML елементами
+ */
+const taskInput = document.querySelector('.task-input');
+const taskList = document.querySelector('.collection');
+const clearBtn = document.querySelector('.clear-tasks');
+const filter = document.querySelector('.filter-input');
+const form = document.querySelector('.create-task-form');
+
+/**
+ * Створюємо слухачі на необхідні нам події
+ */
+document.addEventListener('DOMContentLoaded', renderTasks);
+clearBtn.addEventListener('click', clearAllTasks);
+taskList.addEventListener('click', handleTaskActions);
+form.addEventListener('submit', createTask);
+
+/**
+ * Отримуємо дані з localStorage
+ * @return {[String]} - масив з задачами, або пустий масив, якщо localStorage пустий
+ */
+function getTasksFromLocalStorage() {
+  return localStorage.getItem('tasks') !== null
+    ? JSON.parse(localStorage.getItem('tasks'))
+    : [];
+}
+
+/**
+ * Записуємо дані в localStorage
+ * @param {Array} tasks - масив з задачами
+ */
+function setTasksToLocalStorage(tasks) {
+  localStorage.setItem('tasks', JSON.stringify(tasks));
+}
+
+/**
+ * Створюємо окрему задачу
+ * @param {String} task - окрема задача
+ * @param {Number} taskId - унікальний ідентифікатор завдання
+ */
+function createSingleTaskElement(task, taskId) {
+  // Створюємо HTML елемент li
+  const li = document.createElement('li');
+  // Додаємо елементу клас
+  li.className = 'collection-item';
+  // Кладемо в нього текстову ноду з задачею
+  li.appendChild(document.createTextNode(task));
+
+  // Створюємо обгортку для іконки по кліку на яку буде видалена окрема задача
+  const deleteElement = document.createElement('span');
+  // Додаємо елементу клас
+  deleteElement.className = 'delete-item';
+  // Кладемо в нього іконку
+  deleteElement.innerHTML = '<i class="fa fa-remove"></i>';
+  // Додаємо елемент в елемент списку
+  li.appendChild(deleteElement);
+
+  // Створюємо обгортку для іконки по кліку на яку буде редагування окремої задачі
+  const editElement = document.createElement('span');
+  // Додаємо елементу клас
+  editElement.className = 'edit-item';
+  // Кладемо в нього іконку
+  editElement.innerHTML = '<i class="fa fa-edit"></i>';
+  // Додаємо атрибут data-task-id з унікальним ідентифікатором завдання
+  editElement.setAttribute('data-task-id', taskId);
+  // Додаємо елемент в елемент списку
+  li.appendChild(editElement);
+
+  // Додаємо елемент списку в список завдань
+  taskList.appendChild(li);
+}
+
+/**
+ * Додаємо в DOM існуючі задачі
+ */
+function renderTasks() {
+  // Отримуємо задачі з localStorage або пустий масив
+  const tasks = getTasksFromLocalStorage();
+
+  // Проходимо по масиву задач і додаємо кожну задачу в список, в DOM
+  tasks.forEach((task) => {
+    createSingleTaskElement(task.text, task.id);
+  });
```

Comment:
```suggestion
  tasks.forEach(({ text, id }) => {
    createSingleTaskElement(text, id);
  });
```
Можна використати деструктуризацію і записати так